saas/fsm/suspicious_users/fsm-suspicious_users.pt

```diff
@@ -0,0 +1,149 @@
+name "SaaS Manager - Suspicious Users"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will raise an alert when Flexera SaaS Manager identifies suspicious users logging into SaaS applications. See the [README](https://github.com/flexera/policy_templates/tree/master/saas/fsm/suspicious_users/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
product doesn't want to use alert.  instead change **raise an alert** to **create an incident**.